.github/workflows/tiledb-cloud-py.yaml

```diff
@@ -37,6 +37,7 @@ jobs:
         run: pre-commit run -a -v
 
   run-tests:
+    name: "${{ matrix.tiledb-version-spec }}:${{ matrix.python-version }}:${{ matrix.dependencies }}:${{ matrix.os }}"
```

Comment:
So that the tiledb version appears first in the limited width left sidebar on, for example, https://github.com/TileDB-Inc/TileDB-Cloud-Py/actions/runs/12937321533.